hack/setup-kind.sh

```diff
@@ -3,6 +3,13 @@
 # Giant Swarm CRDs
 kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/giantswarm/apiextensions/15836a106059cc8d201e1237adf44aec340bbab6/helm/crds-common/templates/giantswarm.yaml
 
+# KEDA CRDs for policy testing
+kubectl create --context kind-kyverno-cluster -f https://github.com/kedacore/keda/releases/download/v2.16.1/keda-2.16.1-core.yaml || \
+kubectl create --context kind-kyverno-cluster -f https://github.com/kedacore/keda/releases/download/v2.16.1/keda-2.16.1.yaml
```

Comment:
Why do we have this `or` condition here ?
Should both manifests be deployed ?

Reply:
It's more like if the first one doesn't work (because they might have dropped it at some point) use the second one. But I guess I can just keep one :)